pgsql/pointcloud.sql.in

```diff
@@ -198,6 +198,12 @@ CREATE OR REPLACE FUNCTION PC_EnvelopeAsBinary(p pcpatch)
 	RETURNS bytea AS 'MODULE_PATHNAME', 'pcpatch_envelope_as_bytea'
 	LANGUAGE 'c' IMMUTABLE STRICT;
 
+-- PC_Envelope is an alias to PC_EnvelopeAsBinary. It is kept for
+-- backward-compatibility reasons.
+CREATE OR REPLACE FUNCTION PC_Envelope(p pcpatch)
+	RETURNS bytea AS 'MODULE_PATHNAME', 'pcpatch_envelope_as_bytea'
+	LANGUAGE 'c' IMMUTABLE STRICT;
+
```

Comment:
Could it issue a deprecation warning/error when used, to prepare the switch to a geometry return value in 2.0?